src/testing/README.md

```diff
@@ -1,22 +1,41 @@
 # Module Testing
 
-* this section will be used to hold the information from `best-practices/errtesting` when that section inevitably outgrows a single recipe format (which is a priority)
-* [Crypto Collectables Testing](https://www.shawntabrizi.com/substrate-collectables-workshop/#/5/introduction). 
+Although the Rust compiler ensures safe memory management, it cannot formally verify the correctness of a program's logic. Fortunately, Rust also comes with a convenient suite for writing unit and integration tests. When you initiate code with Cargo, test scaffolding is automatically generated to simplify the developer experience. Testing concepts and syntax are covered in depth in [Chapter 11 of the Rust Book](https://doc.rust-lang.org/book/ch11-00-testing.html).
 
-Things to cover not covered in that example include
+* [Mock Runtime for Unit Testing](./mock.md)
 
-* simple tests at bottom of file `=>` `mock.rs` + `test.rs` via `new_test_ext()` `=>` `struct ExtBuilder`
-* fuzzing
-* benchmarking
-* mocking
+There's also more rigorous testing systems ranging from mocking and fuzzing to formal verification. See [quickcheck](https://docs.rs/quickcheck/0.9.0/quickcheck/) for an example of a property-based testing framework ported from Haskell to Rust.
 
-*I will revisit this section once I have done some of the bottom three in other samples*
+## Kitchen Unit Test Coverage
 
-<!-- ### Old Section
+Unit tests have been written for the following kitchen modules:
+- [`adding-machine`](https://github.com/substrate-developer-hub/recipes/tree/master/kitchen/modules/adding-machine)
```

Comment:
Before this is merged, I want to categorize this by the style of test scaffolding from most configurable and least usable to least configurable and most easy to use.

Reply:
done in most recent commits